staging/docker-registry/templates/statefulset.yaml

```diff
@@ -99,9 +99,21 @@ spec:
       {{- if .Values.nodeSelector }}
       nodeSelector: {{ toYaml .Values.nodeSelector | nindent 8 }}
       {{- end }}
-      {{- if .Values.affinity }}
-      affinity: {{ toYaml .Values.affinity | nindent 8 }}
+      affinity:
+      {{- if .Values.nodeAffinity }}
+        nodeAffinity: {{ toYaml .Values.nodeAffinity | nindent 10 }}
       {{- end }}
+        podAntiAffinity:
```

Comment:
You don't think we can get away without this change and just leave it to the client to set the `affinity` values directly? This chart already exposes `.Values.podLabels` that can be used to select.

Making this change deviates us further from the upstream chart and removes existing ability to set `podAffinity`